Develop

prabhant opened the pull request.

HeidiSeibold commented:
Thanks @jonmcalder ! :cake:

jonmcalder commented:
Sure - happy to help! Will be keen to help out some more in future but I'm very new to OpenML (just signed up last week in order to start exploring) so need to take some time to get familiar with things first. Hopefully I'll be able to make some more contributions along the way as and when I spot opportunities to do so.

sahithyaravi committed:
Update .travis.yml

sahithyaravi committed:
Update .travis.yml

sahithyaravi committed:
Update .travis.yml

sahithyaravi committed:
dash testing

sahithyaravi committed:
Merge branch 'ci' of https://github.com/openml/openml.org into ci

sahithyaravi committed:
add more dependencies

sahithyaravi committed:
add chrome driver

sahithyaravi committed:
Update .travis.yml

sahithyaravi committed:
Update .travis.yml

sahithyaravi committed:
Update .travis.yml

sahithyaravi committed:
Update .travis.yml

sahithyaravi committed:
Update .travis.yml

sahithyaravi committed:
Update .travis.yml

sahithyaravi committed:
Update .travis.yml

prabhant committed:
bcypt base

prabhant committed:
argon2 bcrypt final switch

sahithyaravi committed:
add installation chrome

sahithyaravi committed:
Merge branch 'ci' of https://github.com/openml/openml.org into ci

# Conflicts:
#	.travis.yml

sahithyaravi committed:
update travis yml

sahithyaravi committed:
Update .travis.yml

sahithyaravi committed:
Update .travis.yml

sahithyaravi committed:
Update .travis.yml

sahithyaravi committed:
Update .travis.yml

sahithyaravi committed:
Update .travis.yml

sahithyaravi committed:
add memoize and fix graphs

sahithyaravi committed:
update requirements

sahithyaravi committed:
Merge branch 'develop' into ci

# Conflicts:
#	autoapp.py
#	requirements.txt

sahithyaravi committed:
update travis yml

prabhant committed:
changes- argon2 , mail and reset logic

prabhant committed:
Merge remote-tracking branch 'origin/develop' into develop

prabhant committed:
reset edits

prabhant committed:
eod edits

prabhant committed:
confirmation mail base

prabhant committed:
Confirmation final

prabhant committed:
logout

prabhant committed:
bcrypt fixes

PGijsbers committed:
Merge branch 'develop' into StudyVis

PGijsbers committed:
Add back study after merge conflict.

PGijsbers committed:
Add Flask-Caching as it is required in the development branch but forgotten.

PGijsbers committed:
Merge pull request #100 from openml/StudyVis

Study vis

prabhant committed:
update todos

prabhant committed:
Merge remote-tracking branch 'origin/develop' into develop

sahithyaravi committed:
Update .travis.yml

sahithyaravi committed:
Update .travis.yml

sahithyaravi committed:
Update .travis.yml

sahithyaravi committed:
remove all dataset test

sahithyaravi committed:
Merge branch 'ci' of https://github.com/openml/openml.org into ci

sahithyaravi committed:
Merge branch 'develop' into ci

# Conflicts:
#	autoapp.py
#	requirements.txt
#	server/utils.py

sahithyaravi committed:
merge conflict fix

sahithyaravi committed:
more conflict fix

prabhant committed:
changing JSON requests

sahithyaravi committed:
data tests working

sahithyaravi committed:
working tests

sahithyaravi committed:
update requirements

sahithyaravi committed:
config

sahithyaravi committed:
edit config

sahithyaravi committed:
change numpy and scipy for python 3.8

sahithyaravi committed:
3.8 errors fix

sahithyaravi committed:
python 3.8 doesnt work, todo later

prabhant committed:
env changes

prabhant committed:
env changes

joaquinvanschoren committed:
Merge pull request #101 from openml/develop

Develop

joaquinvanschoren committed:
typo fix

prabhant committed:
correction

joaquinvanschoren committed:
npm audit fix

joaquinvanschoren committed:
clean up dependencies

joaquinvanschoren committed:
env

joaquinvanschoren committed:
cleanup

sahithyaravi committed:
fix warnings sqlite

prabhant committed:
env changes

sahithyaravi committed:
fix sqlite warning

sahithyaravi committed:
flask cache

sahithyaravi committed:
Merge branch 'master' into develop

sahithyaravi committed:
Merge pull request #105 from openml/develop

Develop

joaquinvanschoren committed:
preparing for deploy

prabhant committed:
env changes

prabhant committed:
Merge remote-tracking branch 'origin/master'

prabhant committed:
url correction

prabhant committed:
profile js changes

joaquinvanschoren committed:
clean up requirements

joaquinvanschoren committed:
clean up requirements

prabhant committed:
react env + req.txt edits

sahithyaravi committed:
change to dev for develop

sahithyaravi committed:
Merge branch 'develop' into ci

# Conflicts:
#	requirements.txt

sahithyaravi committed:
try flask run

sahithyaravi committed:
move to bg

sahithyaravi committed:
remove impute

sahithyaravi committed:
admin access

sahithyaravi committed:
Revert "admin access"

This reverts commit 3a64292b36ed56c3672e2358ef93b4a91dc1dd86.

sahithyaravi committed:
fixes

sahithyaravi committed:
remove pkill

sahithyaravi committed:
Merge pull request #106 from openml/ci

Contitinous integration - dash pytest only

prabhant committed:
changes

App style, making factory oriented, sign in edits, deleting tests, requirement configurations.

joaquinvanschoren committed:
moved some environment variables

joaquinvanschoren committed:
Merge pull request #107 from openml/develop

Develop

sahithyaravi committed:
change logging

sahithyaravi committed:
change logging

sahithyaravi committed:
restructure code

sahithyaravi committed:
restructure code

sahithyaravi committed:
working version after restructure #1

sahithyaravi committed:
working version after restructure #1

sahithyaravi committed:
clean up layouts

sahithyaravi committed:
clean up layouts

sahithyaravi committed:
np warning

sahithyaravi committed:
np warning

prabhant committed:
tests first commit

Commit:
tests first commit

Commit:
Merge branch 'master' into develop

prabhant committed:
Merge branch 'master' into develop

sahithyaravi committed:
flake8 fixes and setup 

* flake8

* flake8 errors

* flake8 fixes

* more fixes

* fix more errors

* fix asserts

* change max len

* fix long lines

* fix caching

sahithyaravi committed:
flake8 fixes and setup 

* flake8

* flake8 errors

* flake8 fixes

* more fixes

* fix more errors

* fix asserts

* change max len

* fix long lines

* fix caching

sahithyaravi committed:
dist plot fix

sahithyaravi committed:
dist plot fix

Commit:
test and restructuring

prabhant committed:
test and restructuring

Commit:
Merge remote-tracking branch 'origin/develop' into develop

prabhant committed:
Merge remote-tracking branch 'origin/develop' into develop

sahithyaravi committed:
Merge branch 'develop' of https://github.com/openml/openml.org into develop

sahithyaravi committed:
Merge branch 'develop' of https://github.com/openml/openml.org into develop

sahithyaravi committed:
overview tests

sahithyaravi committed:
overview tests

sahithyaravi committed:
restructure task callback

sahithyaravi committed:
restructure task callback

Commit:
Auto stash before merge of "develop" and "origin/develop"

prabhant committed:
Auto stash before merge of "develop" and "origin/develop"

Commit:
test basics

prabhant committed:
test basics

Commit:
test additions

prabhant committed:
test additions

sahithyaravi committed:
Merge branch 'develop' of https://github.com/openml/openml.org into develop

sahithyaravi committed:
Merge branch 'develop' of https://github.com/openml/openml.org into develop

prabhant committed:
tests

Commit:
tests

prabhant committed:
tests

Commit:
tests

Commit:
tests


tests

prabhant committed:
tests


tests

prabhant committed:
Merge remote-tracking branch 'origin/develop' into develop

Commit:
Merge remote-tracking branch 'origin/develop' into develop

prabhant committed:
Auto stash before merge of "develop" and "origin/develop"

Commit:
Auto stash before merge of "develop" and "origin/develop"

prabhant committed:
edit

Commit:
edit

Commit:
isort based formatting

Formatted code based on isort python library

prabhant committed:
isort based formatting

Formatted code based on isort python library

Commit:
test + script

prabhant committed:
test + script

sahithyaravi committed:
Merge branch 'develop' of https://github.com/openml/openml.org into develop

# Conflicts:
#	server/src/dashboard/data_callbacks.py
#	server/src/dashboard/task_callbacks.py

sahithyaravi committed:
Merge branch 'develop' of https://github.com/openml/openml.org into develop

# Conflicts:
#	server/src/dashboard/data_callbacks.py
#	server/src/dashboard/task_callbacks.py

sahithyaravi committed:
fix flows

sahithyaravi committed:
fix flows

sahithyaravi committed:
more fixes

sahithyaravi committed:
more fixes

sahithyaravi committed:
fix test data

sahithyaravi committed:
fix test data

sahithyaravi committed:
fix tasks

sahithyaravi committed:
fix tasks

sahithyaravi committed:
pep8

sahithyaravi committed:
pep8

sahithyaravi committed:
add dash config (+1 squashed commits)

Squashed commits:

[bb5dfeb] config dash

sahithyaravi committed:
add dash config (+1 squashed commits)

Squashed commits:

[bb5dfeb] config dash

prabhant committed:
Edit profile react functionality

Commit:
Edit profile react functionality

Commit:
Merge remote-tracking branch 'origin/develop' into develop

prabhant committed:
Merge remote-tracking branch 'origin/develop' into develop

sahithyaravi committed:
ci warnings

sahithyaravi committed:
ci warnings

Commit:
API Key page

prabhant committed:
API Key page

prabhant committed:
Merge remote-tracking branch 'origin/develop' into develop

Commit:
Merge remote-tracking branch 'origin/develop' into develop

sahithyaravi committed:
more tests

sahithyaravi committed:
more tests

sahithyaravi committed:
Merge branch 'develop' of https://github.com/openml/openml.org into develop

sahithyaravi committed:
Merge branch 'develop' of https://github.com/openml/openml.org into develop

prabhant committed:
reset api key changes

Commit:
reset api key changes

prabhant committed:
Merge remote-tracking branch 'origin/develop' into develop

Commit:
Merge remote-tracking branch 'origin/develop' into develop

prabhant committed:
literal change

Commit:
literal change

Commit:
Image upload pre

prabhant committed:
Image upload pre

sahithyaravi committed:
fix error in some datasets (+1 squashed commits)

Squashed commits:

[f798cdf] testing

sahithyaravi committed:
fix error in some datasets (+1 squashed commits)

Squashed commits:

[f798cdf] testing

sahithyaravi committed:
more fixes (+1 squashed commits)

Squashed commits:

[60e0dba] restructuring

sahithyaravi committed:
more fixes (+1 squashed commits)

Squashed commits:

[60e0dba] restructuring

sahithyaravi committed:
pep8

sahithyaravi committed:
pep8

Commit:
user confirmation testing

prabhant committed:
user confirmation testing

Commit:
Merge remote-tracking branch 'origin/develop' into develop

prabhant committed:
Merge remote-tracking branch 'origin/develop' into develop

prabhant committed:
api key route testing

Commit:
api key route testing

Commit:
api key route testing


api key testing

prabhant committed:
api key route testing


api key testing

prabhant committed:
Merge remote-tracking branch 'origin/develop' into develop

Commit:
Merge remote-tracking branch 'origin/develop' into develop

Commit:
Send activation token again

prabhant committed:
Send activation token again

Commit:
test additions and travis

prabhant committed:
test additions and travis

prabhant committed:
code refactor

Commit:
code refactor

Commit:
code refactor

prabhant committed:
code refactor

prabhant committed:
code refactor


code refactor

Commit:
code refactor


code refactor

Commit:
Merge remote-tracking branch 'origin/develop' into develop

prabhant committed:
Merge remote-tracking branch 'origin/develop' into develop

prabhant committed:
isort refactor

Commit:
isort refactor

prabhant committed:
image js

Commit:
image js

prabhant committed:
User already exists commit

Commit:
User already exists commit

prabhant committed:
form validation signup

Commit:
form validation signup

prabhant committed:
profile email edit and validation

Commit:
profile email edit and validation

prabhant committed:
Comments docstrings

Commit:
Comments docstrings

prabhant committed:
edits and refactoring

Commit:
edits and refactoring

prabhant committed:
TODOs and test config

Commit:
TODOs and test config

prabhant committed:
Updates

Commit:
Updates

prabhant committed:
Changes

Commit:
Changes

Commit:
changes

prabhant committed:
changes

Commit:
travis removing backend tests

prabhant committed:
travis removing backend tests

Commit:
additions

prabhant committed:
additions

Commit:
reformat

prabhant committed:
reformat

prabhant committed:
request commit

prabhant committed:
request commit

prabhant committed:
Merge remote-tracking branch 'origin/develop' into develop

Commit:
Merge remote-tracking branch 'origin/develop' into develop

Commit:
Merge remote-tracking branch 'origin/develop' into develop

Commit:
Image backend

prabhant committed:
changes

prabhant committed:
Auto stash before revert of "Image backend"

prabhant committed:
Image saving code

prabhant committed:
eod image

sahithyaravi committed:
Update dash_config.py
